src/componant/Films.js

```diff
@@ -43,10 +44,10 @@ export default function Films(){
             value={searchFilm}
             onChange={handelChange}
             />
-            <div>
+            <div className="container-film">
                 {(error)?'error on search'
                 :(searchFilmValue)?
-                searchFilmValue.map((x,index)=><div key={index}>{x.original_title}</div>):'display all films'}
+                searchFilmValue.map((x,index)=><Card key={index} item={x}/>):'display all films'}
```

Comment:
you should id as the key instead of index
```suggestion
                searchFilmValue.map((film)=><Card key={film.id} item={film}/>):'display all films'}
```